.github/workflows/release-against-charts.yml

```
@@ -0,0 +1,64 @@
+name: Release Fleet against rancher/charts
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "Checkout Fleet branch/tag"
+        required: true
+        default: "master"
+      charts_ref:
+        description: "Submit PR against the following rancher/charts branch (e.g. dev-v2.7)"
+        required: true
+        default: "dev-v2.7"
+      prev_fleet:
+        description: "Previous Fleet version (e.g. 0.6.0-rc.3)"
+        required: true
+        default: ""
+      new_fleet:
+        description: "New Fleet version"
+        required: true
+        default: ""
+      prev_chart:
+        description: "Previous Rancher Chart version (e.g. 101.1.0)"
+        required: true
+        default: ""
+      new_chart:
+        description: "New Rancher Chart version"
+        required: true
+        default: ""
+      should_replace:
+        description: "Should the old Fleet version be replaced/removed? (e.g. true in case of release candidate bumps)"
+        required: true
+        default: "true"
+
+jobs:
+  rancher-integration:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{github.event.inputs.ref}}
+          path: fleet
+      - name: Checkout rancher/charts
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          repository: rancher/charts
+          ref: ${{github.event.inputs.charts_ref}}
+          path: charts
+      - name: Run release script
+        run: ./fleet/.github/scripts/release-against-charts.sh ${{github.event.inputs.prev_fleet}} ${{github.event.inputs.new_fleet}} ${{github.event.inputs.prev_chart}}  ${{github.event.inputs.new_chart}}  ${{github.event.inputs.should_replace}}
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{secrets.CHART_PR_PAT}}
+          push-to-fork: thardeck/charts
+          title: ' Update Fleet to v${{github.event.inputs.new_fleet}}'
```

```suggestion
          title: 'Update Fleet to v${{github.event.inputs.new_fleet}}'
```
Ah, that whitespace is probably required because they prefix the version?